### PR TITLE
LSR-16855 jumphost fix

### PR DESCRIPTION
**Pull request by ls-hector**
Removes newrelic and rsyslog

### CODE REVIEW

File: Policyfile.lock.json [hunk 1]
```diff
@@ -60,25 +48,25 @@
...
+        "version": "12.3.4"
       }
     },
     "cron": {
-      "version": "6.4.0",
-      "identifier": "c112ad00b72022a61c427b3deedef49b3a7e043d",
-      "dotted_decimal_identifier": "54345204755800098.46755917995044574.268947538445373",
-      "cache_key": "cron-6.4.0-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/cron/versions/6.4.0/download",
+      "version": "7.0.4",
```
Comment (ls-michielrensen) [inline Policyfile.lock.json:62]: I remember that we had to version-pin this to: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L36

File: Policyfile.lock.json [hunk 2]
```diff
@@ -104,36 +92,36 @@
...
+        "version": "8.0.0"
       }
     },
     "logrotate": {
-      "version": "2.3.0",
-      "identifier": "75c4e3e2c86d65f88e5fea6ac7ad97eaff009185",
-      "dotted_decimal_identifier": "33149055316553061.69962336830146477.167035556368773",
-      "cache_key": "logrotate-2.3.0-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/logrotate/versions/2.3.0/download",
+      "version": "3.0.9",
```
Comment (ls-michielrensen) [inline Policyfile.lock.json:106]: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L38

File: Policyfile.lock.json [hunk 3]
```diff
@@ -104,36 +92,36 @@
...
+        "version": "3.0.9"
       }
     },
     "ntp": {
-      "version": "3.9.1",
-      "identifier": "f72eda0ee8474e5cebe3172f7039ee95c1d12f12",
-      "dotted_decimal_identifier": "69575833335383886.26155058436468793.262326969249554",
-      "cache_key": "ntp-3.9.1-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/ntp/versions/3.9.1/download",
+      "version": "3.12.2",
```
Comment (ls-michielrensen) [inline Policyfile.lock.json:117]: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L41

File: Policyfile.lock.json [hunk 4]
```diff
@@ -181,36 +169,25 @@
       }
     },
     "seven_zip": {
-      "version": "3.2.0",
-      "identifier": "d374d8fece863d7e18d17afdceb5a5f28db7ca76",
-      "dotted_decimal_identifier": "59519695422654013.35493135056228021.182461178301046",
-      "cache_key": "seven_zip-3.2.0-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/seven_zip/versions/3.2.0/download",
+      "version": "4.2.2",
```
Comment (ls-michielrensen) [inline Policyfile.lock.json:172]: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L45

File: Policyfile.lock.json [hunk 5]
```diff
@@ -181,36 +169,25 @@
...
+        "version": "4.2.2"
       }
     },
     "mingw": {
-      "version": "2.1.1",
-      "identifier": "2f7f0e1ca530683eabf54e504b44807b16dc5ce1",
-      "dotted_decimal_identifier": "13369022492258408.17640518625282884.141266152873185",
-      "cache_key": "mingw-2.1.1-supermarket.chef.io",
-      "origin": "https://supermarket.chef.io:443/api/v1/cookbooks/mingw/versions/2.1.1/download",
+      "version": "2.1.4",
```
Comment (ls-michielrensen) [inline Policyfile.lock.json:183]: https://github.com/lightspeedretail/kitchen-camel/blob/6585d41f197e1507ab44f185338b9902b22b3809/policies/use1prod/_default.rb#L39

**Review by ls-michielrensen — APPROVED** [review]
Not sure if it may pose any issues down the line, but we have a couple version pins that are not set in this cookbooks policyfile, not entirely if they are ever used anywhere else outside of it's own test-kitchen scope. Apart from that, lgtm

**Comment by ls-hector** [comment]
While this cookbook does not enforce the version locks you mentioned, they are enforced at the policy level. For example `logrotate` is pinned to version 2.2.2 instead of using the version 3.0.9 specified by `common_linux`. While this might cause trouble when the version in the common cookbook is not compatible with Chef 12, which is not the case here.

**Pull request merged**